docs/docs/packaging/profiles.md

```diff
@@ -82,7 +82,7 @@ target: <string>
 type: <type>
 
 # Default value
-default: <default>
```

Comment:
I believe that using `<any>` is too generic for this case. Instead, we can use OR types as shown [here](https://github.com/NethermindEth/el-node-spec/blob/main/docs/docs/packaging/profiles.md?plain=1#L142). For instance:
```
<int | float | bool | string>
```
This avoids including the `object` type, for which we do not have any available options. Additionally, we can add a comment to clarify that the option must be of the same type

Reply:
Well noticed! Done [b053262](https://github.com/NethermindEth/el-node-spec/pull/52/commits/b0532622ad3f0b19a2ab01cc118060067e81427b)

Reply:
What do you think about the comment clarifying that the option must be the same type? Do you think it is not necessary?

Reply:
I think that is implicit, we are showing examples for each type and all of them fit on the 4 mentioned yaml types.

Reply:
It's ok then